examples/FsharpExamples/Program.fs

```diff
@@ -19,6 +19,7 @@ let main _ =
     PulsarClient.Logger <- loggerFactory.CreateLogger("PulsarLogger") 
 
     runSimple().Wait()
+    runTlsAuthentication().Wait()
```

Comment:
Please comment this line as well as lines below, so only Simple will be run by default